CHANGELOG.md

```diff
@@ -1,10 +1,24 @@
 ## 2.3.0
 
+### Breaking changes
+
+* `RealmResults.distinct()` returns a new `RealmResults` object instead of filtering on the original object (#2947).
```

Comment:
Well technically it was filed as a bug a very long time ago here https://github.com/realm/realm-java/issues/2947 but is probably also the cause of https://github.com/realm/realm-java/issues/2282 which is also a bug